test/evergreen.yml

```diff
@@ -216,16 +216,18 @@ functions:
           set -o errexit
           set -o verbose
 
-          # Check whether specific branches have been provided to the function
-          if [ -z ${branches} ]; then
-            wt_branches=${branch_name}
+          # Check whether specific branches have been provided to the function through the task
+          # variable defined in doc-update. If none have been specified, the built-in Evergreen
```

Comment:
If none have been specified, the built-in Evergreen expansion ${branch_name} is used --> If none are specified, use the built-in Evergreen expansion ${branch_name}

Reply:
Thanks, corrected in ed593ccc050d835888324ecda4396e4f18c41316